### DISCUSSION TITLE
Questions about the Cross and Change operator

### DISCUSSION THREAD

**Discussion by ghost** [statnet/tergm#78] [Q&A] [answered]
Is there a precise mathematical theory about the Cross and Change operators. The Form and Diss operators are consistent with the paper which proposed STERGM. However, my understanding of the Cross and Change operators is not clear. Is the Change operator consistent with the paper which proposed TERGM? So what exactly does the cross operator represent?

**Comment by krivit** [comment, marked as answer]
That particular formulation goes back to Hanneke and Xing (2007), Hanneke, Fu, and Xing (2010), and Krackhardt and Handcock (2007), as well as the [`btergm` package](https://www.jstatsoft.org/article/view/v083i06). We also discuss it and provide examples in the [`tergm` tutorial](http://statnet.org/Workshops/tergm/tergm_tutorial.html).
